pkgs/by-name/op/openhue-cli/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  writableTmpDirAsHomeHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "openhue-cli";
+  version = "0.18";
+
+  src = fetchFromGitHub {
+    owner = "openhue";
+    repo = "openhue-cli";
+    tag = finalAttrs.version;
+    hash = "sha256-yIPqjKIvYk2Y9BYieqrm5QvvAHnuImXEDbI1JOy0kBA=";
+  };
+
+  vendorHash = "sha256-lqIzmtFtkfrJSrpic79Is0yGpnLUysPQLn2lp/Mh+u4=";
+
+  env.CGO_ENABLED = 0;
+
+  rev = "c37f2a910173d8c9df42b40bd6efb583307a40cc";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${finalAttrs.version}"
+    "-X main.commit=${finalAttrs.rev}"
```

```suggestion

  ldflags = [
    "-s"
    "-w"
    "-X main.version=${finalAttrs.version}"
    "-X main.commit=${finalAttrs.version}"
```

I suggest adding this as `rev` will not be automatically updated

---

The downside is that this is used to generate a URL to the commit, appending this to `https://github.com/openhue/openhue-cli/commit/`, if I change it to version that won't work anymore. I haven't found a way to get the commit sha of the tag as a simple variable or even with a pre-existing function, but do you know if something like that exists?